util/merkle/benches/triedb.rs

```diff
@@ -139,9 +139,11 @@ fn bench_read_multiple(b: &mut Bencher) {
 fn bench_write_single(b: &mut Bencher) {
     let mut db = TestDB::new(DB_SIZE);
     b.iter(|| {
-        let mut trie = db.trie_mut();
-        let item = random::<usize>() % DB_SIZE + DB_SIZE;
-        let _ = trie.insert(&item.to_be_bytes(), &item.to_be_bytes()).unwrap();
+        {
```

Comment:
Why did you add a new block?

Reply:
To drop trie before flushing db.

Reply:
Both `trie_mut` and `flush` mutably borrow `db`.

Reply:
It seems it's not related to the commit title "Do not export TrieDB and TrieDBMut".
Could you extract it as a new commit?

Reply:
It's related. The return type of `trie_mut` was not bounded to the lifetime of `db`, but this patch makes it bounded.

Reply:
Oh, I thought that there would be no compile error even if we flatten the block.
It seems we need it.